autoload.php

```diff
@@ -70,6 +69,7 @@
         "Gaia\\Core\\MVC\\REST\\Controllers" => APP_PATH. '/core/mvc/controllers/',
         "Gaia\\Core\\MVC\\Models\\Relationships\\Factory" => APP_PATH. '/core/mvc/models/relationships/factory',
         "Gaia\\Core\\MVC\\Models\\Query" => APP_PATH. '/core/mvc/models/query/',
+        "Tests\\Gaia\\Acl" => APP_PATH. '/tests/acl/'
```

Comment:
I think the Autoload for tests should be seperate.

Reply:
Created a separate autoloader for testing purpose